adam-core/src/main/scala/org/bdgenomics/adam/rich/RichAlignmentRecord.scala

```diff
@@ -68,7 +68,7 @@ class IlluminaOptics(val tile: Long, val x: Long, val y: Long) {}
 
 class RichAlignmentRecord(val record: AlignmentRecord) {
 
-  lazy val referenceLength: Int = RichAlignmentRecord.referenceLengthFromCigar(record.getCigar.toString)
+  lazy val referenceLength: Int = RichAlignmentRecord.referenceLengthFromCigar(record.getCigar)
```

Comment:
Thanks for getting these; we've got several redundant `toString`s hanging around from the pre-[bdg-formats/36](https://github.com/bigdatagenomics/bdg-formats/pull/36) merged.